### PR TITLE
Catalog categories updates for new console landing page

### PR DESCRIPTION
**Pull request by spadgett**
@ahardin-rh This change probably needs to be in the 3.7 upgrade doc since it affects existing extensions that customize categories. Please let me know how I can add that.

### CODE REVIEW

**Comment by spadgett** [comment]
@ahardin-rh PTAL. These changes were for 3.7.

**Review by ahardin-rh — APPROVED** [review]
LGTM! As for adding this to the upgrade docs, I just discussed this with @adellape who is currently making some changes to the Upgrade content. You can create a PR against the master branch in the Upgrade section with the content you want to add, then we can get it into the 3.7 doc set. Thanks!

**Comment by spadgett** [comment]
@ahardin-rh thanks!

@jwforres ok with the changes?

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -663,58 +663,55 @@ this customization in future upgrades if you modify existing category items.
...
+  // Required.
   label: "Go",
-  // Optional.  If specified, defines a unique icon for this item
-  iconClass: "font-icon icon-go-gopher",
-  // Optional.  If specified, enables matching other tag values to this category
-  // item
-  categoryAliases: [
+  // Optional. If specified, defines a unique icon for this item.
+  icon: "icon-go-gopher",
+  // Required. Items with matching any tag will appear in this subcategory.
```
Comment (jwforres) [inline install_config/web_console_customization.adoc]: this sentence doesnt read right to me, probably should have been "Items matching any"

same sentence appears again a couple times below

**Comment by spadgett** [comment]
Thanks, updated

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/web_console_customization.adoc#install-config-web-console-customization[Customizing the Web Console]
|Updated the xref:../install_config/web_console_customization.adoc#configuring-catalog-categories[Configuring Catalog Categories] section to reflect the latest web console categories.
%

**Pull request merged**